Dockerfile

```diff
@@ -212,6 +213,19 @@ RUN mkdir -p /vmtoolsd/${LIBDNET} &&\
 RUN cd $ROOTFS && cd usr/local/lib && ln -s libdnet.1 libdumbnet.so.1 &&\
     cd $ROOTFS && ln -s lib lib64
 
+# Build XenServer Tools
+ENV XENTOOLS_REPO xe-guest-utilities
+ENV XENTOOLS_BRANCH boot2docker
+
+RUN cd / && \
+    git clone https://github.com/xenserver/$XENTOOLS_REPO && \
+    cd $XENTOOLS_REPO && \
+    git checkout $XENTOOLS_BRANCH && \
```

Comment:
Thanks. I've updated this PR at https://github.com/phusl/boot2docker/commit/ecb760364b27593374e0824235c82dcbdb26dfd3